### DISCUSSION TITLE
Change the order of events

### DISCUSSION THREAD

**Discussion by Lion1302** [nhn/tui.calendar#1245] [Q&A] [answered]
Hello,
Is there any way to place every event one under the other? For instance, I have four events at 9 am. So, I'd like to have four rows with the title '9:00 am' and each event is under the other one instead of having them side by side. 
PS. Duration of an event do not matter to me.
![example](https://user-images.githubusercontent.com/70651345/183735714-197dd4e0-814a-4cd0-b9bc-b07c3dd88323.png)

**Comment by adhrinae** [comment, marked as answer]
Sadly, We don't provide this kind of customization.

**Comment by Lion1302** [comment]
Thank you for answering.
